socorro/external/es/index_creator.py

```diff
@@ -45,8 +40,10 @@ class IndexCreator(RequiredConfig):
     required_config.elasticsearch.add_option(
         'shards_per_index',
         default=10,
-        doc='number of shards to set in newly created indices. Elasticsearch '
-            'default is 5.',
+        doc=(
+            'number of shards to set in newly created indices. Elasticsearch '
+            'default is 5.'
+        )
```

Comment:
The parens make it easier for me to read this as "one string" rather than multiple strings. That's a me thing.